### ISSUE TITLE
Dashboard Page refresh ==> signout

### ISSUE THREAD

**Issue by gkorland** [omnistrate/saasbuilder#23]
![image](https://github.com/omnistrate/saasbuilder/assets/753206/4f8bbc17-1a68-4ec6-8948-a09fc2116b0f)

**Comment by sharmaamits** [comment]
We are unable to reproduce this issue, could you please share the reproducible steps or the actions your performed before encountering this error, to get more context.

**Comment by gkorland** [comment]
yes, it happens each time, also happened today to @swilly22, on every refresh it logs out

**Comment by sharmaamits** [comment]
@gkorland this shouldn't happen now
